src/environmentd/src/http/sql.rs

```diff
@@ -1279,6 +1278,29 @@ async fn execute_stmt<S: ResultSender>(
             ),
             ctx_extra,
         },
+        ExecuteResponse::Copying { rx, .. } => {
+            let num_rows = match sender.await_rows(client.canceled(), rx).await? {
+                Ok(CopyToResponseUnary::Rows(rows)) => {
```

Comment:
cc @teskje I know we talked about removing the `CopyToResponseUnary` and reusing `CopyToResponse`, but here in environmentd `CopyToResponse` is not available as it does not depend upon compute-client

Reply:
done